locales/cmd/cluster/common/active.en.toml

```diff
@@ -3,8 +3,7 @@ one = 'Location of the kubeconfig file.'
 
 [cluster.common.flag.offline.token.description]
 one = '''
-Provide an offline token to be used by the operator.
-To get a token, visit https://cloud.redhat.com/openshift/token.
+Provide an offline token to be used by the operator. To obtain a token, visit {{.OfflineTokenURL}}.
```

Comment:
Moved to a one-liner because the second line was not actually showing in the help of the command.